src/app/timer-bar/timer-bar.component.scss

```diff
@@ -0,0 +1,37 @@
+// @import "@material/linear-progress/mdc-linear-progress.scss";
+
+$bufferColor: #A1A1A1;
+
+mdc-linear-progress {
+    height: 40px;
+    border-radius: 16px;
+}
+
+// The only way I've found to change the buffer color on the progress bar
+::ng-deep .mdc-linear-progress__buffer {
```

Comment:
`::ng-deep` is [deprecated](https://angular.io/guide/deprecations#deep-component-style-selector)